src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/ConnectionString/DbConnectionStringKeywords.cs

```diff
@@ -0,0 +1,72 @@
+        internal const string ConnectTimeout = "Connect Timeout";
+        internal const string ContextConnection = "Context Connection";
+        internal const string CurrentLanguage = "Current Language";
+        internal const string DataSource = "Data Source";
+        internal const string EnclaveAttestationUrl = "Enclave Attestation Url";
+        internal const string Encrypt = "Encrypt";
+        internal const string Enlist = "Enlist";
+        internal const string FailoverPartner = "Failover Partner";
+        internal const string FailoverPartnerSPN = "Failover Partner SPN";
+        internal const string HostNameInCertificate = "Host Name In Certificate";
+        internal const string InitialCatalog = "Initial Catalog";
+        internal const string IntegratedSecurity = "Integrated Security";
+        internal const string IPAddressPreference = "IP Address Preference";
+        internal const string LoadBalanceTimeout = "Load Balance Timeout";
+        internal const string MaxPoolSize = "Max Pool Size";
+        internal const string MinPoolSize = "Min Pool Size";
+        internal const string MultipleActiveResultSets = "Multiple Active Result Sets";
+        internal const string MultiSubnetFailover = "Multi Subnet Failover";
+        internal const string NetworkLibrary = "Network Library";
+        internal const string PacketSize = "Packet Size";
+        internal const string Password = "Password";
+        internal const string PersistSecurityInfo = "Persist Security Info";
+        internal const string PoolBlockingPeriod = "Pool Blocking Period";
+        internal const string Pooling = "Pooling";
+        internal const string Replication = "Replication";
+        internal const string ServerCertificate = "Server Certificate";
+        internal const string ServerSPN = "Server SPN";
+        internal const string TransactionBinding = "Transaction Binding";
+        internal const string TransparentNetworkIPResolution = "Transparent Network IP Resolution";
+        internal const string TrustServerCertificate = "Trust Server Certificate";
+        internal const string TypeSystemVersion = "Type System Version";
+        internal const string UserID = "User ID";
+        internal const string UserInstance = "User Instance";
+        internal const string WorkstationID = "Workstation ID";
+        
+        #if NETFRAMEWORK
+        // Odbc
+        internal const string Driver = "Driver";
+        internal const string Dsn = "Dsn";
+        internal const string FileDsn = "FileDsn";
+        internal const string SaveFile = "SaveFile";
+
+        // OleDb
+        internal const string FileName = "File Name";
+        internal const string OleDbServices = "OLE DB Services";
+        internal const string Provider = "Provider";
+
+        // OracleClient
+        internal const string Unicode = "Unicode";
+        internal const string OmitOracleConnectionName = "Omit Oracle Connection Name";
+        #endif
```

Comment:
All of these except for `Driver` could be removed altogether, they're never referenced.

`Driver` is also technically only referenced in a dead code path in DbConnectionOptions, but might be a little out of scope.

Reply:
Hmm. That's a valid point... I think the origin of this file was to basically be a mirror of the keywords from System.Data.Common. Otherwise, why would we have oracle nonsense in here?
In the cleanup PR I have queued, I'm getting rid of all but the one you mentioned that is still around.